scripts/components/Nav.js

```diff
@@ -12,6 +13,7 @@ class Nav extends Component {
         super(props);
         this.login = this.login.bind(this);
         this.logout = this.logout.bind(this);
+        this.toggleStats = this.toggleStats.bind(this);
```

Comment:
I just followed the way you did this with login/logout.  Not entirely sure what's going on here, but 'm guessing this is just a way of binding functions to this with es6 classes.